<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CHNMCT369326L" failures="0" tests="6" name="com.testCases.MobileFlow" time="30.855" errors="1" timestamp="31 Jul 2020 07:21:23 GMT" skipped="5">
  <testcase classname="com.testCases.MobileFlow" name="TC_004_OppoMobileFunctionality" time="17.374">
    <skipped/>
  </testcase> <!-- TC_004_OppoMobileFunctionality -->
  <testcase classname="com.testCases.MobileFlow" name="beforeMethod" time="13.293">
    <error type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
at com.amazon.selenium.base.SeleniumBase.startApp(SeleniumBase.java:277)
at com.qa.ApplicationSpecifics.ProjectSpecificMethods.beforeMethod(ProjectSpecificMethods.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeMethod -->
  <testcase classname="com.testCases.MobileFlow" name="TC_005_LGMobileFunctionality" time="0.000">
    <skipped/>
  </testcase> <!-- TC_005_LGMobileFunctionality -->
  <testcase classname="com.testCases.MobileFlow" name="TC_006_RealmeMobileFunctionality" time="0.000">
    <skipped/>
  </testcase> <!-- TC_006_RealmeMobileFunctionality -->
  <testcase classname="com.testCases.MobileFlow" name="runScenario" time="0.000">
    <skipped/>
  </testcase> <!-- runScenario -->
  <testcase classname="com.testCases.MobileFlow" name="runScenario" time="0.188">
    <skipped/>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.testCases.MobileFlow -->
